msg/messages.js

```diff
@@ -24,6 +24,8 @@
  *
  * After modifying this file, run `npm run translate` from the root directory
  * to regenerate `./msg/json/en.json`.
+ * IMPORTANT:
```

Comment:
This file and the message files are now permanent merge conflicts with blockly.  You'll need to add them to `cleanup.sh` to make sure they're resolved correctly.

Currently `cleanup.sh` just removes (almost) all of the files in `msg/js` and `msg/json`.  You'll need to update it to resolve in favor of the scratch-blocks file.  That code is ~[here](https://github.com/LLK/scratch-blocks/blob/develop/cleanup.sh#L21).

You'll also need to add `msg/messages.js` to the explicit list of files that should always be resolved in favor of scratch-blocks.  That's [here](https://github.com/LLK/scratch-blocks/blob/develop/cleanup.sh#L80).

Reply:
I'm actually debating whether we really want to check in the js and json files. Either way, I'll fix up the cleanup script. Thanks for pointing it out.